codespell_lib/data/dictionary.txt

```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
 aautomatically->automatically
 abailable->available
 abanden->abandon
+abandenment->abandonment
+abandining->abandoning
+abandomnent->abandonment
+abandond->abandoned
 abandonded->abandoned
+abandonding->abandoning
 abandone->abandon
+abandonig->abandoning
+abandonne->abandonment, abandon,
 abandonned->abandoned
+abandonnent->abandonment
 abandonning->abandoning
+abanonded->abandoned
 abbbreviated->abbreviated
 abberation->aberration
 abberations->aberrations
 abberivates->abbreviates
+abberivation->abbreviation
+abberivations->abbreviations
 abberration->aberration
+abberrations->aberrations
+abberviation->abbreviation
+abberviations->abbreviations
 abbort->abort, abbot,
 abborted->aborted
 abborting->aborting
 abborts->aborts, abbots,
 abbout->about, abbot,
+abbreivation->abbreviation
 abbrevate->abbreviate
 abbrevated->abbreviated
 abbrevation->abbreviation
 abbrevations->abbreviations
+abbreveation->abbreviation
+abbreveations->abbreviations
+abbreviatin->abbreviation, abbreviating,
+abbreviatins->abbreviations
+abbreviato->abbreviation
 abbreviaton->abbreviation
 abbreviatons->abbreviations
+abbrievation->abbreviation
+abbrievations->abbreviation
 abbriviate->abbreviate
 abbriviation->abbreviation
 abbriviations->abbreviations
+abdominable->abdominal
```

Comment:
Perhaps `abdominable->abdominal, abominable,` although the second is certainly less probable?